README.md

```diff
@@ -1,5 +1,9 @@
 ![](https://img.shields.io/badge/STATUS-NOT%20CURRENTLY%20MAINTAINED-red.svg?longCache=true&style=flat)
-
+################################################################################
```

Comment:
This section is now prominently displayed in caption style when somebody reads the README on GitHub. Better use a .reuse/dep5 file to annotate files like the README.